packages/in_app_purchase/in_app_purchase_storekit/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.5
```

Comment:
@stuartmorgan I bumped the iOS federated version by a minor version for the iOS minimum change, wdyt?

Reply:
Seems reasonable. There's a lot of grey area in minor vs bugfix.